packages/create-react-app/index.js

```diff
@@ -230,6 +231,24 @@ function getPackageName(installPackage) {
   return installPackage;
 }
 
+function checkNpmVersion() {
+  // Check npm version
+  var npmVersionProc = spawn.sync('npm', ['--version']),
+    npmVersion = npmVersionProc.stdout;
+
+  if (npmVersion) {
+    var recommendVersion = semver.lt(npmVersion.toString(), '3.0.0');
+    if (recommendVersion) {
+      console.log(
+        chalk.green(
```

Comment:
Can you make this yellow instead? I think we can remove the "Tip:" prefix.
Formatting wise, I would prefer

```js
console.log(chalk.yellow(...));
console.log(chalk.yellow(...));
```

for each line and then another

```js
console.log();
```

for a new line at the very end.

Thanks!